libr/core/cmd_search.c

```diff
@@ -2716,7 +2723,7 @@ static int cmd_search(void *data, const char *input) {
 		if (input[0] == 'j') {
 			json = true;
 		}
-	/* pass-thru */
+		// fallthrough
```

Comment:
https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html -Wimplicit-fallthrough=n

`fallthrough` is recognized but not `pass-thru`.